scene/gui/tree.cpp

```diff
@@ -166,6 +166,20 @@ TreeItem::TreeCellMode TreeItem::get_cell_mode(int p_column) const {
 	return cells[p_column].mode;
 }
 
+/* multiline */
+void TreeItem::set_multiline(int p_column, bool p_multiline) {
+	ERR_FAIL_INDEX(p_column, cells.size());
+
+	cells.write[p_column].multiline = p_multiline;
+
+	_changed_notify(p_column);
```

Comment:
I'd remove the empty lines. Separating everything into single-line blocks looks cluttered.

Reply:
Done.